<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Color_Picker_Widget_UI</class>
 <widget class="QWidget" name="Color_Picker_Widget_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>562</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="Main_Layout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="Display_Button_Layout">
       <item>
        <widget class="QPushButton" name="Color_Wheel_Button">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>色轮</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RGB_Button">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>RGB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HSV_Button">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>HSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Swatches_Button">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>常用</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Color_Wheel" name="Color_Wheel_Widget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="RGB_Layout" native="true">
       <layout class="QVBoxLayout" name="RGB_Layout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="R_Layout" stretch="0,0,1">
          <item>
           <widget class="QLabel" name="R_Label">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="R_LineEdit">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>D00</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Color_Slider" name="R_Slider" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="G_Layout" stretch="0,0,1">
          <item>
           <widget class="QLabel" name="G_Label">
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="G_LineEdit">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>D00</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Color_Slider" name="G_Slider" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777211</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="B_Layout" stretch="0,0,1">
          <item>
           <widget class="QLabel" name="B_Label">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="B_LineEdit">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>D00</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Color_Slider" name="B_Slider" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="HSV_Layout" native="true">
       <layout class="QVBoxLayout" name="HSV_Layout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="H_Layout" stretch="0,0,1">
          <item>
           <widget class="QLabel" name="H_Label">
            <property name="text">
             <string>H</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="H_LineEdit">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>D00</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Color_Slider" name="H_Slider" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="S_Layout" stretch="0,0,1">
          <item>
           <widget class="QLabel" name="S_Label">
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="S_LineEdit">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>D00</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Color_Slider" name="S_Slider" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="V_Layout" stretch="0,0,1">
          <item>
           <widget class="QLabel" name="V_Label">
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="V_LineEdit">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>D00</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Color_Slider" name="V_Slider" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Color_Wheel</class>
   <extends>QWidget</extends>
   <header>Color_Wheel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Color_Slider</class>
   <extends>QWidget</extends>
   <header>Color_Slider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
